app/forms/post_form.py

```diff
@@ -20,7 +22,11 @@ class PostForm(FlaskForm):
         'Caption',
         render_kw={'rows': 5, 'style': 'height: auto'},
         validators=[
-            DataRequired(message='Caption Required')
+            DataRequired(message='Caption Required'),
+            Length(
+                max=DESCRIPTION_LENGTH_MAX,
+                message=f"Caption must be less than {DESCRIPTION_LENGTH_MAX} characters long."
```

Comment:
Max value is inclusive in validation. Message needs to be change to something like `Caption cannot be more than {DESCRIPTION_LENGTH_MAX} characters long.`

Reply:
Sounds good i'll update